src/mailer/engine.py

```diff
@@ -156,15 +168,21 @@ def _require_no_backend_loop(mailer_email_backend):
 def send_all():
     """
     Send all eligible messages in the queue.
+
+    The actual backend to use for sending, defaulting to the Django default.
+    To make testing easier this is not stored at module level.
```

Comment:
This does not belong in the docstring of method `send_all`. Please revert that change. If that's causing CI issues, please find a different fix with better semantics.